contracts/protocol/facets/OrchestrationHandlerFacet.sol

```diff
@@ -33,6 +35,44 @@ contract OrchestrationHandlerFacet is
         DiamondLib.addSupportedInterface(type(IBosonOrchestrationHandler).interfaceId);
     }
 
+    /**
+     * @notice Raises a dispute and immediately escalates it.
+     *
+     * Caller must send (or for ERC20, approve the transfer of) the
+     * buyer escalation deposit percentage of the offer price, which
+     * will be added to the pot for resolution.
+     *
+     * Emits a DisputeEscalated event if successful.
+     *
+     * Reverts if:
+     * - The disputes region of protocol is paused
```

Comment:
Please add:
```suggestion
     * - The disputes region of protocol is paused
     * - The orchestration region of protocol is paused
```
```

Reply:
sorry for not being clearer, but it's missing from IBosonOrchestationHandler.sol as well